scene/register_scene_types.cpp

```diff
@@ -936,10 +936,10 @@ void register_scene_types() {
 	OS::get_singleton()->yield(); //may take time to init
 
 	for (int i = 0; i < 20; i++) {
-		GLOBAL_DEF("layer_names/2d_render/layer_" + itos(i + 1), "");
-		GLOBAL_DEF("layer_names/2d_physics/layer_" + itos(i + 1), "");
-		GLOBAL_DEF("layer_names/3d_render/layer_" + itos(i + 1), "");
-		GLOBAL_DEF("layer_names/3d_physics/layer_" + itos(i + 1), "");
+		GLOBAL_DEF(vformat("layer_names/2d_render/layer_%d", i), "");
+		GLOBAL_DEF(vformat("layer_names/2d_physics/layer_%d", i), "");
+		GLOBAL_DEF(vformat("layer_names/3d_render/layer_%d", i), "");
+		GLOBAL_DEF(vformat("layer_names/3d_physics/layer_%d", i), "");
```

Comment:
Maybe we can use 2-digits padding so that things have the proper order in the docs? i.e. `layer_00`, `layer_01`, etc.

Reply:
Alternatively we could update the docs logic to use natural number comparison instead of lexographic.

Reply:
Can't we fix this by using natural sort in the documentation and property editors?

Reply:
Looks good. We should just fix the XML generator then so that it also looks good for doc writers (but that's not something that needs to be done in this PR).